package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "badge",
+  "version": "0.1.0",
+  "description": "Badge",
+  "scripts": {
+    "test": "karma start --instrumenter && node tests/remap.js"
```

Comment:
Missing ` --single-run` parameter. This is need for CI server
When running locally continuously use `karma start`